datacatalog/cloud-client/src/test/java/com/example/datacatalog/CreateEntryTests.java

```diff
@@ -62,45 +62,67 @@ public void tearDown() {
   @AfterClass
   public static void tearDownClass() {
     try (DataCatalogClient dataCatalogClient = DataCatalogClient.create()) {
-      dataCatalogClient.deleteEntryGroup(
-          EntryGroupName.of(PROJECT_ID, LOCATION, ENTRY_GROUP_ID_NO_CHILDREN).toString());
-
-      dataCatalogClient.deleteEntry(
-          EntryName.of(PROJECT_ID, LOCATION, PARENT_ENTRY_GROUP_ID, ENTRY_ID).toString());
-      dataCatalogClient.deleteEntryGroup(
-          EntryGroupName.of(PROJECT_ID, LOCATION, PARENT_ENTRY_GROUP_ID).toString());
+      // Must delete Entries before deleting the Entry Group.
+      if (entriesPendingDeletion.isEmpty() || entryGroupsPendingDeletion.isEmpty()) {
+        fail("Something went wrong, no entries were generated");
+      }
+
+      for (String entryName : entriesPendingDeletion) {
+        dataCatalogClient.deleteEntry(entryName);
+      }
+
+      for (String entryGroupName : entryGroupsPendingDeletion) {
+        dataCatalogClient.deleteEntryGroup(entryGroupName);
+      }
     } catch (Exception e) {
       System.out.println("Error in cleaning up test data:\n" + e.toString());
     }
   }
 
   @Test
   public void testCreateFilesetEntry() {
+    String entryGroupId = "fileset_entry_group_parent_" + getUuid8Chars();
+    String entryId = "fileset_entry_id_" + getUuid8Chars();
+
     // Must create a Entry Group before creating the entry.
-    CreateEntryGroup.createEntryGroup(PROJECT_ID, PARENT_ENTRY_GROUP_ID);
-    CreateFilesetEntry.createEntry(PROJECT_ID, PARENT_ENTRY_GROUP_ID, ENTRY_ID);
+    CreateEntryGroup.createEntryGroup(PROJECT_ID, entryGroupId);
+    CreateFilesetEntry.createEntry(PROJECT_ID, entryGroupId, entryId);
+
+    // Store names for clean up on teardown
+    String expectedEntryGroupName =
+        EntryGroupName.of(PROJECT_ID, LOCATION, entryGroupId).toString();
```

Comment:
FYI: I think these helpers are being deprecated. At least, they were deprecated in Python (https://github.com/googleapis/gapic-generator/issues/3012) and the same reasoning applies to the Java generator (it's not possible with new proto annotations for configuring the generator).

Reply:
Do you know the recommended way?
I see two alternatives...

1. Use EntryGroupName.Builder....setProject(project).setLocation(location).setEntryGroup(entryGroup).build()
2. Build the string manually

 Thanks

Reply:
At least in Python (2) is recommended, as the resource builders are going away there. I haven't seen any discussion about it outside of Python, though.

I don't know why, as the same configuration changes affect all languages.

Reply:
I see, it makes sense going that way if this configuration will affect all languages... 
@kurtisvg Do you know in Java the recommended way?

Reply:
AFAIK, they aren't going away for Java. They are generally preferred to manually formatting the string.

Reply:
@tswast WDYT can we keep the helpers?

Reply:
I have doubts that they will actually stay in Java, based on my conversations with Luke on the ACtools team. I'll check with him if the resource helper deprecation is supposed to affect more than just Python.

Reply:
Sure, will wait for your confirmation.

Reply:
You can proceed with using the resource builders. I misunderstood the reasoning for removal in Python.

Reply:
so AFAIK the PR is ready to be merged, @tswast are you able to review it and merge if everything is ok?